wandb/sdk/artifacts/storage_handlers/gcs_handler.py

```diff
@@ -188,14 +189,14 @@ def _entry_from_obj(
         return ArtifactManifestEntry(
             path=posix_name,
             ref=URIStr(f"{self._scheme}://{str(posix_ref)}"),
-            digest=obj.md5_hash,
+            digest=obj.etag,
             size=obj.size,
             extra=self._extra_from_obj(obj),
         )
 
     @staticmethod
     def _extra_from_obj(obj: "gcs_module.blob.Blob") -> Dict[str, str]:
-        return {
-            "etag": obj.etag,
-            "versionID": obj.generation,
-        }
+        extra = {"versionID": obj.generation}
```

Comment:
(question): just curious, why do we save the md5 in `extra` here?

Reply:
There might not actually be any good reason to. Before we stored the etag and I followed that pattern and swapped them, but on reflection I'm not sure there's any utility in it.